sdks/go/pkg/beam/core/runtime/graphx/user.go

```diff
@@ -80,6 +81,28 @@ func DecodeFn(data string) (reflectx.Func, error) {
 	return fn.Fn, nil
 }
 
+// EncodeGraphFn encodes a *graph.Fn as a string.
+func EncodeGraphFn(u *graph.Fn) (string, error) {
```

Comment:
Are we not using these new helpers anywhere? The PR title suggests we're just making something existing public.

Reply:
It's related to Combiner Lifting, so I've used that jira tag.
Since it can help with writing a bundle level precombine, using existing combineFns, which is what I'm using it for at present, since the beam model/SDK doesn't yet support lifting combines through a CoGroupByKey. That's the root of it anyway. I like thinking of it as enabling the writing of MetaStructs.

The main goal is for me to avoid needing to write my precombine helper one way now and then re-writing later when there is official support for such a thing. If that feature does show up, I can simply use the same CombineFns over again.

Thanks!